### ISSUE TITLE
Dark Icons

### ISSUE THREAD

**Issue by bsudekum** [mapbox/mapbox-directions.js#35]
It'd be great to have a dark icon set as well.

cc @samanpwbb

**Comment by samanpwbb** [comment]
@rsudekum if you are work on labs projects, you can use base to access dark icons (https://www.mapbox.com/base/styling/icons/). I can add an inverted theme here as well - might be a couple days before I get to it though.

**Comment by bsudekum** [comment]
@samanpwbb yup checked out using the base icons. The only issue is that the class names do not line up. Fine with waiting a couple of days.
